config/config_test.go

```diff
@@ -86,6 +120,19 @@ func TestConfigS3_EmptyRules(t *testing.T) {
 	return
 }
 
+func TestConfigIamRole_EmptyRules(t *testing.T) {
+	configFilePath := "./mocks/iamRole_empty_rules.yaml"
+	configObj, err := GetConfig(configFilePath)
+
+	require.NoError(t, err)
+
+	if !reflect.DeepEqual(configObj, emptyConfig()) {
+		assert.Fail(t, "Config should be empty, %+v\n", configObj)
+	}
+
+	return
```

Comment:
NIT: not needed?

I just realized some other test functions that were here before you have this for some reason... Probably something we need to clean up!